packages/design-system-docs/src/styles/components/_details.scss

```diff
@@ -31,10 +31,6 @@
     @extend .ds-text;
   }
 
-  > p:not([class]):first-child {
```

Comment:
<img width="747" alt="Screen Shot 2020-07-28 at 11 53 08 AM" src="https://user-images.githubusercontent.com/5424713/88696528-fde68100-d0c8-11ea-88ce-ac6f3f2f01d4.png">
This is what it looks like now. Previously the first `p` element would be larger because it would be styled as a `lead`